de-lib/src/main/java/org/iplantc/de/desktop/client/views/windows/SystemMessagesWindow.java

```diff
@@ -20,9 +22,16 @@
 
     private MessagesPresenter presenter;
 
+
     @Inject
-    SystemMessagesWindow(final IplantDisplayStrings displayStrings) {
+    SystemMessagesWindow(final IplantDisplayStrings displayStrings,
+                         final UserInfo userInfo) {
+        this.userInfo = userInfo;
         setHeading(displayStrings.systemMessagesLabel());
+        String width = getSavedWidth(WindowType.SYSTEM_MESSAGES.toString());
+        String height = getSavedHeight(WindowType.SYSTEM_MESSAGES.toString());
+        setSize((width == null) ? computeDefaultWidth() + "" : width, (height == null) ? computeDefaultHeight() + "" : height);
```

Comment:
It looks like the `setWidth` and `setHeight` calls below will override this `setSize` call?

Reply:
Yup those needs to removed! Sorry Oversight on my part!